From 29296ca9464433668463852d04bd879db3d9d4ab Mon Sep 17 00:00:00 2001
From: Yongxin Liu <yongxin.liu@windriver.com>
Date: Thu, 13 Jun 2024 10:26:35 +0000
Subject: [PATCH 10/24] rdk/snr: klm/cpk/ice_fltr.h: use int for
 ice_fltr_add_mac_to_list

kernel commit 5e24d5984c80 (ice: Use int for ice_status).

Upstream-Status: Inappropriate [RDK is released in tarball only]

Signed-off-by: Yongxin Liu <yongxin.liu@windriver.com>
---
 klm/cpk/ice_fltr.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/klm/cpk/ice_fltr.h b/klm/cpk/ice_fltr.h
index 5929dfa..df7e4fb 100644
--- a/klm/cpk/ice_fltr.h
+++ b/klm/cpk/ice_fltr.h
@@ -6,7 +6,7 @@
 
 void ice_fltr_free_list(struct device *dev, struct list_head *h);
 
-enum ice_status
+int
 ice_fltr_add_mac_to_list(struct ice_vsi *vsi, struct list_head *list,
 			 const u8 *macaddr, enum ice_sw_fwd_act_type action);
 enum ice_status ice_fltr_add_mac(struct ice_vsi *vsi, const u8 *mac,
-- 
2.39.3

